docs/smart-contracts/guides/dapps/Terminal-UI-Guide.mdx

```diff
@@ -0,0 +1,1603 @@
+# Guide: Building Functional User Interfaces for Dapps using Terminal User Interface (TUI)
+
+## Table of Contents
```

Comment:
Do we need the TOC here since it's displayed in the right-hand nav bar? The TOC seems to clog things up a bit so I might remove it.

Reply:
👍🏻 for removing

Reply:
Also, for the sake of completeness, Docusaurus can provide [inline TOCs for you](https://docusaurus.io/docs/2.x/markdown-features/toc#inline-table-of-contents). We don't really use it anywhere, but it's probably an easier way to make it happen, and keep up-to-date over time.